src/Microsoft.Azure.WebJobs.Host/Scale/ScaleMonitorService.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Hosting;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Threading;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using System.Linq;
+using Microsoft.Extensions.Options;
+using Microsoft.Azure.WebJobs.Hosting;
+
+namespace Microsoft.Azure.WebJobs.Host.Scale
+{
+    /// <summary>
+    /// Class to store scale metrics.
```

Comment:
This type is being moved from Functions Host. I don't know why you randomly changed the class xml doc to something less correct from what it already was https://github.com/Azure/azure-functions-host/blob/8c8d2df01b6a3d5dbfbf5e268241eef8d4f4eb83/src/WebJobs.Script/Scale/FunctionsScaleMonitorService.cs#L20 which is more correct.